features/kernel/src/main/resources/features.xml

```diff
@@ -0,0 +1,84 @@
+<!--
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+ -->
+<features xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          name="${project.artifactId}-${project.version}"
+          xmlns="http://karaf.apache.org/xmlns/features/v1.3.0"
+          xsi:schemaLocation="http://karaf.apache.org/xmlns/features/v1.3.0 http://karaf.apache.org/xmlns/features/v1.3.0">
+
+    <repository>mvn:org.apache.karaf.features/standard/${karaf.version}/xml/features</repository>
+    <repository>mvn:org.apache.karaf.features/spring/${karaf.version}/xml/features</repository>
+
+    <feature name="kernel" version="${project.version}"
+             description="Minimual set of features and dependencies commonly used.">
+        <feature version="${karaf.version}">standard</feature>
+        <feature version="${karaf.version}">eventadmin</feature>
+        <feature version="${karaf.version}">jasypt-encryption</feature>
+        <feature version="${karaf.version}">http</feature>
+    </feature>
+
+    <feature name="javax-validation" version="${project.version}">
+        <bundle>mvn:javax.validation/validation-api/${validation.version}</bundle>
+    </feature>
+
+    <feature name="joda-converter" version="${project.version}">
```

Comment:
The downside to this is we would increase integration test time since each feature file has it's own test starting it's own container . If we can find a way to mitigate that, I'm all for it.